src/spanningTree/spanningTree_process.cpp

```
@@ -79,8 +79,8 @@ void pgr_process_spanningTree(
 
             switch (val) {
                 case 1:
-                    if (distance < 0) {
-                        pgr_throw_error("Negative value found on 'distance'", "Must be non negative");
+                    if (distance <= 0) {
+                        pgr_throw_error("Invalid value of 'distance'", "Valid values are greater than 0");
```

Cppcheck (2.19.0)

[style] The function 'pgr_free' is never used.

(unusedFunction)